### DISCUSSION TITLE
Welcome to AREPL-vscode Discussions!

### DISCUSSION THREAD

**Discussion by Almenon** [Almenon/AREPL-vscode#369] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mami1963** [comment]
import pyautogui
import time
import sys

# ANSI escape kodları ile renk ve arka plan ayarları
WHITE_BACKGROUND = '\033[47m'  # Beyaz arka plan
GREEN_TEXT = '\033[32m'  # Yeşil yazı
RESET = '\033[0m'  # Renk sıfırlama

def sağ_tıklama_yap(coordinate, label):
    try:
        pyautogui.rightClick(coordinate)
        print(f"{GREEN_TEXT}{label} sağ tıklama yapıldı: {coordinate}{RESET}")
    except Exception as e:
        print(f"{GREEN_TEXT}Bir hata oluştu: {e}{RESET}")

def main():
    # Başlangıçta sol tıklama yapılacak koordinat
    print(f"{WHITE_BACKGROUND}{GREEN_TEXT}Başlangıç sol tıklama yapılıyor...{RESET}")
    pyautogui.leftClick(578, 225)
    print(f"{GREEN_TEXT}Başlangıç sol tıklama yapıldı: (578, 225){RESET}")

    while True:
        # İlk sağ tıklama (1159, 555)
        sağ_tıklama_yap((1159, 555), "İlk")

        # 20 dakika (1200 saniye) bekle
        print(f"{GREEN_TEXT}20 dakika bekleniyor...{RESET}")
        time.sleep(1200)

        # İkinci sağ tıklama (1226, 443)
        sağ_tıklama_yap((1226, 443), "İkinci")

        # 20 dakika bekle
        print(f"{GREEN_TEXT}20 dakika bekleniyor...{RESET}")
        time.sleep(1200)

        # Üçüncü sağ tıklama (762, 747)
        sağ_tıklama_yap((762, 747), "Üçüncü")

        # 20 dakika bekle
        print(f"{GREEN_TEXT}20 dakika bekleniyor...{RESET}")
        time.sleep(1200)

# Eğer dosya bu modülde çalıştırılmışsa, main() fonksiyonunu başlatıyoruz.
if __name__ == "__main__":
    try:
        main()
    except KeyboardInterrupt:
        print(f"\n{GREEN_TEXT}Program durduruldu. Çıkılıyor...{RESET}")
        sys.exit(0) ben burda hata arıyorum sorun sizce nerde
